xml/System.Threading/ThreadingAclExtensions.xml

```diff
@@ -112,9 +112,9 @@
         <Parameter Name="semaphore" Type="System.Threading.Semaphore" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="semaphore">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="semaphore">The semaphore from which the security descriptors should be returned.</param>
+        <summary>Returns the security descriptors for the specified <paramref name="semaphore" />.</summary>
+        <returns>The security descriptors for the specified <paramref name="semaphore" />.</returns>
```

Comment:
In some return summaries it says "the specified semaphore" (or mutex, etc.), but in others, like here, it paramref's them. I think I might prefer it as just an English word instead of a paramref.

```suggestion
        <returns>The security descriptors for the specified semaphore.</returns>
```

Reply:
Sounds good, thanks